drivers/nrf_radio_802154/CMakeLists.txt

```diff
@@ -74,9 +75,10 @@ zephyr_compile_definitions(
   NRF_802154_CCA_ED_THRESHOLD_DEFAULT=${CONFIG_NRF_802154_CCA_ED_THRESHOLD}
 
   # Disable unused radio driver features
-  NRF_802154_CSMA_CA_ENABLED=0
+  NRF_802154_CSMA_CA_ENABLED=1
+  NRF_802154_TX_STARTED_NOTIFY_ENABLED=1
+
   NRF_802154_ACK_TIMEOUT_ENABLED=0
   NRF_802154_FRAME_TIMESTAMP_ENABLED=0
   NRF_802154_DELAYED_TRX_ENABLED=0
```

Comment:
It seems other features could be enabled now as well. Are we going to do that as a next step?

Reply:
Yes, I just focused on CSMA/CA here.